beta-src/src/components/ui/WDScrollButton.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from "react";
+import Button from "@mui/material/Button";
+import { scrollButtonProps } from "../../interfaces/PhaseScroll";
+import WDPhaseArrowIcon from "../svgr-components/WDPhaseArrowIcon";
+
+const WDScrollButton: React.FC<scrollButtonProps> = function ({
+  onClick = undefined,
+  direction,
+  disabled = false,
+}): React.ReactElement {
+  return (
+    <Button
```

Comment:
can we use WDButton instead?

Reply:
I could, but I would have to add a "direction" prop to WDButton so that it knows which arrow to generate for the scroll button. I didn't want to mess with WDButton so I decided to create my own, but I can make that change then its an easy swap!

Reply:
if you did that, it would be styled already and we could keep them in sync. just make them optional and set default values...then when you use it here you can pass the disableRipple, etc.